src/haz3lcore/statics/Term.re

```diff
@@ -319,6 +319,29 @@ module UPat = {
       }
     };
   };
+
+  let rec get_all_bindings = (pat: t) => {
```

Comment:
technically this can be obtained by just extracting the variables names from the pattern's context. but this is going to change a bit with ADTs, so it's fine to just leave it for now. maybe add a "TODO(andrew)" comment on it for me to remind me to update it to the new co-ctx system once its fully in place.